NETWORK/NetworkGetPresenceInviteSessionId.md

```diff
@@ -6,11 +6,13 @@ aliases: ["0x26E1CD96B0903D60"]
 
 ```c
 // 0x26E1CD96B0903D60 0xC5E0C989
-Any NETWORK_GET_PRESENCE_INVITE_SESSION_ID(Any p0);
+Hash NETWORK_GET_PRESENCE_INVITE_SESSION_ID(int inviteIndex);
+```
+```
+The Native returns a hash of the session id as string from the specific invite index!
```

Comment:
Same as above, please don't make descriptions in code blocks (outlined [here](https://github.com/citizenfx/natives/blob/master/.github/CONTRIBUTING.md)).